src/challenges/challengeOne/3.Table/Table.test.tsx

```diff
@@ -1,8 +1,33 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Table } from "./Table";
-
-// describe("Table", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { Table } from "./Table";
+
+describe("Table", () => {
+  const columns = [
+    { property: "name", label: "Name" },
+    { property: "email", label: "Email" },
+  ];
+
+  const data = [
+    { id: 1, name: "John Doe", email: "johndoe@example.com" },
+    { id: 2, name: "Jane Smith", email: "janesmith@example.com" },
+  ];
+
+  it("renders table with correct columns and data", () => {
```

Comment:
Again you're testing two separate things, the columns render and the data in the table renders, should be two separate tests.